tests/snmpsim/zynos-mes.snmprec

```diff
@@ -0,0 +1,2 @@
+1.3.6.1.2.1.1.1.0|4|MES Something
```

Comment:
Es is because we don't have the data. If you can add the proper sysdescr output we can merge this